src/bitfield/types.py

```diff
@@ -202,13 +202,17 @@ def __getattr__(self, key):
             raise AttributeError('%s is not a valid flag' % key)
         return self.get_bit(self._keys.index(key))
 
+    __getitem__ = __getattr__
```

Comment:
idk why i didnt do this ages ago

Reply:
This change and the related getattr() changes seem like something that could be shipped independently of this change. If you want I could extract those commits out and make a separate pull request out of them.